_posts/2013-02-19-background-image-options.md

```diff
@@ -16,15 +16,18 @@ prefixed: false
   /* various options, see e.g. http://www.css3files.com/background */
   background: url(image1.png), 2 url(image2.png), 3 url(image3.png) #FFF;
```

Comment:
I just noticed these **2** and **3** should probably be removed from `2  url(..)` and `3 url(..)`...

Reply:
Got it!